### PR TITLE
Logo customisation tutorial.

### PR DESCRIPTION
**Pull request by paulkeogh**
A tutorial that explains how to customise the Kaui nav bar logo.

### CODE REVIEW

File: userguide/tutorials/logo_customisation.adoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+i.e. disable the default kaui gem reference and specify explicitly the newly built gem from the killbill-admin-ui
+project.
+
+* Run bundle update
+----
+
+    bundle update
+----
+
+* Run bundle info to verify that the kaui gem is correctly referenced from the killbill-admin-up project
```
Comment (pierre) [inline userguide/tutorials/logo_customisation.adoc:48]: What is `killbill-admin-up`?

**Review by pierre — COMMENTED** [review]
@paulkeogh What is your use-case for customizing the logo?

I'm not sure we want to document these steps that way: because of the Ruby ecosystem, a lot of these steps are not necessarily deterministic and could cause random issues at runtime (plus, you don't want to be running Kaui via `rails s` in production).

Depending on your needs, maybe we can find a better way to white label the UI.

**Comment by paulkeogh** [comment]
I was investigating the ease (or otherwise) of white labelling. I documented a process as it took me a while to figure out, and thought it might be useful to share, but fully agree there's a better way. 

Sorry, that's a typo, should be killbill-admin-ui

**Comment by pierre** [comment]
> I was investigating the ease (or otherwise) of white labelling.

But why do you need white labeling? Are you re-selling Kill Bill? What are you exact requirements? 

Just trying to get a better idea of what you are trying to do and make sure you are on the right track.

**Comment by paulkeogh** [comment]
No plans to resell, evaluating currently and white labelling is one of the criteria for product management guys.

**Comment by pierre** [comment]
> No plans to resell

No problem if you do, but I can think of so many other places where the Kill Bill brand would leak that you'd need to take a holistic approach.

> evaluating currently and white labelling is one of the criteria for product management guys.

If you can share these criteria, feel free to reach out by email. It's a better medium for dicussions.

**Pull request closed without merging**